scripts/draft-release.sh

```diff
@@ -15,7 +15,10 @@ fi
 FOLDER="${2-}"
 
 echo "Drafting release"
-github-release release --user storj --repo storj --tag "$TAG" --draft
+current_major_release_version=$(echo "$TAG"| cut -d '.' -f 1-2)
```

Comment:
This should be major + minor, right?

Reply:
Maybe last_release_version is a little clearer?

Reply:
@stefanbenten renamed to `previous_major_release_version` because `last_release_version ` a bit confusing with next variable `previous_release_version`